pages/opam-packaging.html

```diff
@@ -171,12 +178,10 @@ <h3>Create a package definition file</h3>
 <code>md5=</code> are also supported.)
 </p>
 
-<p>Create a new branch named <code>coq-foo.1.0.0</code>,
-and commit the new <code>opam</code> file:
+<p>Commit the new <code>opam</code> file:
 </p>
 
-<pre><code># Create the branch coq-foo.1.0.0 based on <code>upstream/master</code>
-git checkout -b coq-foo.1.0.0 upstream/master
+<pre><code>
```

Comment:
It seems that there is a spurious newline that shows in the rendered HTML.

Reply:
Whoops, fixed.